test/functional/test-resources.js

```diff
@@ -17,10 +17,15 @@
 import {AmpDocSingle} from '../../src/service/ampdoc-impl';
 import {Resources} from '../../src/service/resources-impl';
 import {Resource, ResourceState} from '../../src/service/resource';
+import {LayerElement} from '../../src/service/layers-impl';
 import {VisibilityState} from '../../src/visibility-state';
 import {layoutRectLtwh} from '../../src/layout-rect';
 import * as sinon from 'sinon';
 
+function stupidMutateLayoutBox(resource, rect) {
```

Comment:
Nope. This is reaching into a private variable in another, unrelated class. I want to make this method go away when we redo the Resource tests.